.github/workflows/announce-release.yaml

```diff
@@ -0,0 +1,20 @@
+name: Send message to Slack when a release is published
+on:
+  release:
+    types: [published]
+
+jobs:
+  slack-send:
+    name: Send message to Slack
+    runs-on: ubuntu-latest
+    steps:
+      - id: slack
+        uses: slackapi/slack-github-action@v1.25.0
+        with:
+          # This data can be any valid JSON from a previous step in the GitHub Action
+          payload: |
+            {
+              "release": "${{ github.event.release.tag_name }} - ${{ github.event.release.html_url }} is going live now!"
+            }
+env:
+  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_ASK_DEVEX_WEBHOOK_URL }}
```

Comment:
I noticed that this secret is named `ASK_DEVEX`, but the body of the PR says it posts to ask-c8-documentation. Which of the two does it post to, and is it the one that you intended?

Reply:
Copy & paste hold over. I can regenerate and edit, so it's not confusing.

The Webhook URL is unique per workflow. I can also make you a maintainer of the workflow on Slack so you can "see" the URL.